ZipBuilder/Sources/FirebaseReleaser/main.swift

```diff
@@ -17,70 +17,53 @@
 import Foundation
 
 import ArgumentParser
-import ManifestReader
+import FirebaseManifest
+import Utils
 
-struct FirebasePodUpdater: ParsableCommand {
+struct FirebaseReleaser: ParsableCommand {
   /// The root of the Firebase git repo.
+  // TODO: Add a default that sets the current repo - ['git', 'rev-parse', '--show-toplevel']
   @Option(help: "The root of the firebase-ios-sdk checked out git repo.",
           transform: URL.init(fileURLWithPath:))
   var gitRoot: URL
 
-  /// A file URL to a textproto with the contents of a `FirebasePod_Release` object. Used to verify
-  /// expected version numbers.
-  @Option(name: .customLong("releasing-pods"),
-          help: "The file path to a textproto file containing all the releasing Pods, of type `FirebasePod_Release`.",
-          transform: URL.init(fileURLWithPath:))
-  var currentRelease: URL
+  @Option(default: false,
+          help: "Log without executing the shell commands")
+  var logOnly: Bool
```

Comment:
I couldn't find any usages of this or any comments as to why this would be used - is this purely for test purposes? If so, that would be helpful to have documented here.

Reply:
Thanks! See #6774